src/engraving/types/constants.h

```diff
@@ -90,6 +90,9 @@ struct Constants
 //       - Corrections to key signature and transposition (#18998)
 //       - New inside/outside style for ties
 
+//    4.30 (Version 4.3)
+//       - New styles for tab notation
+
```

Comment:
I'll PR this comment separately to the master branch, if and when this PR gets merged